js/src/features/project/components/Team/TeamAddMemberForm.tsx

```diff
@@ -0,0 +1,110 @@
+import React, { useCallback, useMemo, useState } from "react";
+import { Button, Form, Select } from "antd";
+
+import { AddProjectTeamMemberRequest, Role } from "ProjectModels";
+import { EmployeeShort } from "HumanResourceModels";
+
+interface Props {
+  teamId: string;
+  roles: Role[];
+  add: (form: AddProjectTeamMemberRequest) => void;
+  employees: EmployeeShort[];
+}
+
+const { Option } = Select;
+
+export const TeamAddMemberForm: React.FC<Props> = ({
+  teamId,
+  roles,
+  add,
+  employees,
+}) => {
+  const [form] = Form.useForm();
+  const [searchResult, setSearchResult] = useState<
+    { label: string; value: string }[]
+  >([]);
+
+  const users = useMemo(
+    () =>
+      employees.map((emp) => ({
+        value: emp.userId,
+        label: `${emp.firstName} ${emp.lastName}`,
+      })),
+    [employees],
+  );
+
+  const onFinish = useCallback(
+    (values) =>
+      add({
+        user: values.user,
+        role: values.role,
+        teamId,
+      }),
+    [add, teamId],
+  );
+
+  const onFinishFailed = useCallback((errorInfo: any) => {
+    console.log("onFinishFailed:", errorInfo);
+  }, []);
+
+  const onSearch = (searchText: string) => {
```

Comment:
Too loooong. You can make it shorter:
```
setSearch(users.filter(u => u.label.includes(searchText.trim())));
```

Reply:
This is shorter, but you need to check for spaces and an empty field, as there are users without a name